opensfm/src/csfm.cc

```diff
@@ -11,13 +11,22 @@
 #include "akaze.cc"
 #include "bundle.h"
 
+#if (PY_VERSION_HEX < 0x03000000)
+static void numpy_import_array_wrapper()
+#else
+static int* numpy_import_array_wrapper()
+#endif
+{
+  /* Initialise numpy API and use 2/3 compatible return */
+  import_array();
+}
 
 BOOST_PYTHON_MODULE(csfm) {
   using namespace boost::python;
   
   google::InitGoogleLogging("csfm");
   boost::python::numeric::array::set_module_and_type("numpy", "ndarray");
-  import_array();
+  numpy_import_array_wrapper();
```

Comment:
BOOST_PYTHON_MODULE declare a function returns `void`
But `import_array()` of numpy 2&3 return different which will cause compile error.
https://github.com/numpy/numpy/blob/master/numpy/core/code_generators/generate_numpy_api.py#L136